windows/tools/OobSimulator/OobSimulatorViewModel.cpp

```diff
@@ -0,0 +1,107 @@
+// clang-format off
+#include "pch.h"
+#include "OobSimulatorViewModel.h"
+#include "OobSimulatorViewModel.g.cpp"
+// clang-format on
+
+#include "Utils.hxx"
+
+#include <nearobject/serialization/UwbSessionDataJsonSerializer.hxx>
+#include <shobjidl.h>
+#include <uwb/protocols/fira/UwbConfigurationBuilder.hxx>
+#include <uwb/protocols/fira/UwbSessionData.hxx>
+
+namespace winrt::OobSimulator::implementation
+{
+OobSimulatorViewModel::OobSimulatorViewModel()
+{
+    m_uwbSessionData = winrt::make<OobSimulator::implementation::UwbSessionData>();
+    m_setUwbSessionData = winrt::make<OobSimulator::implementation::DelegateCommand>(std::bind(&OobSimulatorViewModel::SetUwbSessionData, this));
+
+    // Set default values
+    m_uwbSessionData.SessionId(L"1234");
+    m_uwbSessionData.DeviceRole(OobSimulator::DeviceRole::Responder);
+    m_uwbSessionData.MultiNodeMode(OobSimulator::MultiNodeMode::Unicast);
+    m_uwbSessionData.ControllerMacAddress(L"12:34");
+    m_uwbSessionData.ControleeShortMacAddress(L"67:89");
+}
+
+winrt::OobSimulator::UwbSessionData
+OobSimulatorViewModel::UwbSessionData()
+{
+    return m_uwbSessionData;
+}
+
+winrt::Microsoft::UI::Xaml::Input::ICommand
+OobSimulatorViewModel::SetUwbSessionDataCommand()
+{
+    return m_setUwbSessionData;
+}
+
+winrt::Windows::Foundation::IAsyncAction
+OobSimulatorViewModel::SetUwbSessionData()
+{
+    uwb::protocol::fira::UwbSessionData uwbSessionData;
+    uwb::protocol::fira::UwbConfiguration::Builder builder{};
```

Comment:
nit: Try to keep the way you call the default constructor consistent (either use {} or not)